src/qibolab/result.py

```diff
@@ -77,6 +82,16 @@ def __add__(self, data):
         new_res.std = np.append(self.std, data.std)
         return new_res
 
+    @property
+    def average(self):
+        """Perform average over i and q."""
+        return self
```

Comment:
Maybe an error?

(It's true that is already averaged, but asking for its average doesn't make sense)

Reply:
Just a temporary patch for qibocal.